Tests/LibWeb/Text/expected/wpt-import/css/cssom/shorthand-values.txt

```diff
@@ -2,15 +2,15 @@ Harness status: OK
 
 Found 20 tests
 
-15 Pass
-5 Fail
+14 Pass
+6 Fail
 Pass	The serialization of border: 1px; border-top: 1px; should be canonical.
 Pass	The serialization of border: 1px solid red; should be canonical.
 Pass	The serialization of border: 1px red; should be canonical.
 Pass	The serialization of border: red; should be canonical.
 Fail	The serialization of border-top: 1px; border-right: 1px; border-bottom: 1px; border-left: 1px; border-image: none; should be canonical.
 Fail	The serialization of border-top: 1px; border-right: 1px; border-bottom: 1px; border-left: 1px; should be canonical.
-Pass	The serialization of border-top: 1px; border-right: 2px; border-bottom: 3px; border-left: 4px; should be canonical.
+Fail	The serialization of border-top: 1px; border-right: 2px; border-bottom: 3px; border-left: 4px; should be canonical.
```

Comment:
It looks like this was passing by coincidence previously. This failure can be replicated by just adding the `color-interpolation` property to `Properties.json`.

Reply:
That's really odd. What's the actual result for it now?

Reply:
The test output is:
`assert_equals: expected "border-width: 1px 2px 3px 4px; border-style: none; border-color: currentcolor;" but got "border-style: none; border-width: 1px 2px 3px 4px; border-color: currentcolor;"`

Reply:
I think these should be being sorted into [preferred order](https://drafts.csswg.org/cssom/#concept-shorthands-preferred-order), but it doesn't look like we're doing that?